great_expectations/rule_based_profiler/parameter_builder/metric_parameter_builder.py

```diff
@@ -91,20 +91,20 @@ def _build_parameters(
             "metric_value_kwargs": metric_value_kwargs,
             "metric_dependencies": None,
         }
-        parameter_values: Dict[str, Dict[str, Any]] = {
+        parameter_values: Dict[str, Any] = {
             self.fully_qualified_parameter_name: {
-                "value": validator.get_metric(
+                "val": validator.get_metric(
```

Comment:
Per design review conversation this should change back to `value`
```suggestion
                "value": validator.get_metric(
```

Reply:
This has to be in already... @anthonyburdi ?